app/javascript/dashboard/routes/dashboard/inbox/InboxList.vue

```diff
@@ -94,12 +94,7 @@ const loadMoreNotifications = () => {
 const markNotificationAsRead = async notificationItem => {
   useTrack(INBOX_EVENTS.MARK_NOTIFICATION_AS_READ);
 
-  const {
-    id,
-    primary_actor_id: primaryActorId,
-    primary_actor_type: primaryActorType,
-  } = notificationItem;
-
+  const { id, primaryActorId, primaryActorType } = notificationItem;
```

Comment:
This change fixes an issue in the inbox view context menu where selecting **“Mark as Read”** incorrectly marks all notifications as read.